Vic2ToHoI4/Data_Files/configurables/cultureGroupToGraphics.txt

```diff
@@ -187,6 +187,7 @@ germanic = {
 		neutrality = { "gfx/interface/ideas/idea_albert_kesselring.dds" }
 		radical = { "gfx/interface/ideas/idea_hjalmar_schacht.dds" } 
 	}
+	male_operator_portraits = { "gfx/leaders/GER/portrait_ger_nikolaus_ritter.dds" "gfx/leaders/GER/portrait_ger_otto_skorzeny.dds" "gfx/leaders/SOV/portrait_sov_richard_sorge.dds" }
```

Comment:
This uses ```male_operator_portraits``` some places and ```male_operative_portraits``` others. It should use just one. The converter uses ```male_operative_portraits``` but can be updated for anything.

Reply:
Fixed